lib/cc/engine/markdownlint.rb

```diff
@@ -33,7 +35,7 @@ def include_paths
         return root unless engine_config.has_key?("include_paths")
 
         markdown_files = engine_config["include_paths"].select do |path|
-          path.end_with?(".md")
+          EXTENSIONS.include?(File.extname(path)) || path.end_with?("/")
```

Comment:
FWIW: I'd be in favor of supporting `*.md` only if that's what the underlying tool does and it simplifies the engine in a meaningful way (which it seems like it would). At least until we see actual user demand for `*.markdown` support.